lib/github.ml

```diff
@@ -111,6 +112,11 @@ let gh_link_of_string url_str =
   | Some host ->
   match Re2.find_submatches_exn gh_link_re path with
   | [| _; prefix; Some owner; Some name; Some link_type; Some item |] ->
+    let item =
+      match String.get item (String.length item - 1) with
+      | '/' -> String.sub item ~pos:0 ~len:(String.length item - 1)
+      | _ -> item
+    in
```

Comment:
can probably be simplified. See `Caml.String.ends_with`, `Base.String.is_suffix`, `Base.String.suffix`, `Base.String.chop_suffix_if_exists`.